doc/news/_preparation_next_release.md

```diff
@@ -226,6 +226,9 @@ Many problems were resolved with the following fixes:
 - The [`list` plugin](http://libelektra.org/plugins/list) which is responsible
   for global mounting had a bug which prevented globally mounted plugins from
   being configurable. *(Thomas Wahringer)*
+- Fix an issue with excluded plugins for rest-backend builds with ASAN enabled *(Lukas Winkler)*
```

Comment:
Because the release notes describe the change from one release to another, no "issue with excluded plugins" was fixed (seen over the whole release, ni was never excluded).